Game/Resources/final_event/MainScript.as

```diff
@@ -113,7 +112,7 @@ class MainScript {
 
                 if (fogTimer >= fadeStartTime) {
                     fadeTimer += deltaTime;
-                    float fadeRatio = fadeTimer / fadeApexDuration;
+                    float fadeRatio = fadeTimer / fogApexDuration;
```

Comment:
This should still be `fadeApexDuration`. They apply to different techniques that are interleaved to create the final fade effect.

Reply:
whoops